ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -42,14 +44,23 @@ Item {
         root.invitedCommunity = getCommunity()
     }
 
-   Connections {
-       target: root.store.communitiesModuleInst
-       onCommunityChanged: function (communityId) {
-           if (communityId === root.communityId) {
-               root.invitedCommunity = getCommunity()
-           }
-       }
-   }
+    Connections {
+        target: root.store.communitiesModuleInst
+        onCommunityChanged: function (communityId) {
+            if (communityId === root.communityId) {
+                root.invitedCommunity = getCommunity()
+            }
+        }
+    }
+
+    Connections {
+        target: root.store.communitiesModuleInst
+        onCommunityAdded: function (communityId) {
```

Comment:
@alaibe in these cases you could just add `onCommunityAdded` slot to the previous connections object, that's possible as long as `target` refers to the object that sends the signal you want to connect to.